packaging/setup/ovirt_engine_setup/keycloak/constants.py

```diff
@@ -30,6 +31,7 @@ def _(m):
 class Const(object):
     OVIRT_ENGINE_KEYCLOAK_SETUP_PACKAGE_NAME = \
         'ovirt-engine-keycloak'
+    OVIRT_ENGINE_KEYCLOAK_REALM = 'master'
```

Comment:
Hmm, shouldn't it be `internal` to mimic AAA?

Reply:
Unfortunately, keycloak requires 'super' admin (owning master realm).  
I had in mind creating another admin@internal  or admin@ovirt user that would actually mimic AAA (however, I had not yet come to it).

Reply:
OK, so that needs to be properly documented. I hate those super admins and now we are already going to have 2 (one for grafana and one for keycloak) :-(

Reply:
admin for 'internal' realm will be done in the following PR as next step.